content/07.pathogenesis.md

```diff
@@ -6,11 +6,23 @@ This section would also be great for the introduction of zoonotic diseases which
 
 #### Primary Transmission and Viral Entry
 
+Like that of SARS-CoV, SARS-CoV-2 entry into host cells is mediated by the interaction between the viral spike glycoprotein (S) and human angiotensin-converting enzyme 2 (ACE2) in humans and other animals [@doi:10.1126/science.1116480; @doi:10.1016/j.cell.2020.02.058; @doi:10.1126/science.abb2762; @doi:10.1038/s41586-020-2179-y; @doi:10.1101/2020.02.19.956235; @doi:10.1016/j.cell.2020.02.052; @doi:10.1016/j.cell.2020.03.045; @doi:10.1128/jvi.00127-20].
 ACE2 receptors have been identified as the binding site for SARS-CoV-2 in animals and humans [@doi:10.1038/nature02145; @doi:10.1007/s00134-020-05985-9; @doi:10.1128/JVI.00127-20]. 
```

Comment:
Now that this sentence has been combined with the one above, I think this sentence can be deleted.

Reply:
@anskelly Oh no, sorry I missed this! Do you want to make a PR to fix that and add yourself as an author? I just merged the text in! Let me know if you need help, I got my email back under control so I should see it!

Reply:
@rando2 Thank you! I think I just made another pull request to delete that sentence... I'll try to add myself as an author soon too! Thanks again for all your help -- it's much appreciated! :)

Reply:
Thanks @anskelly! I realized I gave you a hard task to edit two files in the same PR -- if you use this link it should work: <https://github.com/anskelly/covid19-review/blob/patch-2/content/metadata.yaml>